src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumGameOptionPages.java

```diff
@@ -15,29 +16,27 @@
 import me.jellysquid.mods.sodium.client.gui.options.control.TickBoxControl;
 import me.jellysquid.mods.sodium.client.gui.options.storage.MinecraftOptionsStorage;
 import me.jellysquid.mods.sodium.client.gui.options.storage.SodiumOptionsStorage;
-import me.jellysquid.mods.sodium.client.compatibility.workarounds.Workarounds;
 import me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkBuilder;
-import net.minecraft.client.AttackIndicatorStatus;
-import net.minecraft.client.CloudStatus;
-import net.minecraft.client.GraphicsStatus;
-import net.minecraft.client.Minecraft;
-import net.minecraft.client.ParticleStatus;
+import net.minecraft.client.*;
 import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
 import org.lwjgl.opengl.GL;
 import org.lwjgl.opengl.GLCapabilities;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class SodiumGameOptionPages {
-    private static final SodiumOptionsStorage sodiumOpts = new SodiumOptionsStorage();
-    private static final MinecraftOptionsStorage vanillaOpts = new MinecraftOptionsStorage();
+    public static final SodiumOptionsStorage sodiumOpts = new SodiumOptionsStorage();
+    public static final MinecraftOptionsStorage vanillaOpts = new MinecraftOptionsStorage();
```

Comment:
Please expose these as public static getters in this class that return `OptionStorage<SodiumGameOptions>` and the equivalent for vanilla instead. That way we can change the underlying implementation later without a breaking change.